nym-api/src/node_status_api/models.rs

```diff
@@ -315,8 +316,13 @@ impl From<HistoricalUptime> for OldHistoricalUptimeResponse {
 
 // TODO rocket remove smurf name after eliminating `rocket`
 pub(crate) type AxumResult<T> = Result<T, AxumErrorResponse>;
+
+#[derive(ToSchema, ToResponse)]
```

Comment:
why do we need `ToSchema` on this type? it's never directly returned to the user. `status` is used to set correct HTTP status response and the underlying `message` is what reaches the user

Reply:
Because it's listed as a returned type [here](https://github.com/nymtech/nym/blob/develop/nym-api/src/ecash/api_routes/partial_signing.rs#L45) and in many other places like that

Reply:
I could replace it with `String` like this

```rust
responses(
    // ...
    (status = 400, body = String),
)
```